lines.js

```diff
@@ -14,6 +14,24 @@ var historyForward = document.querySelector(".history-button.forward");
 var currentIndex = 0;
 var isOut;
 
+historyBack.buttonAppearance = function() {
```

Comment:
It is nice that you have found this way, but adding properties to `HtmlElement` is really dirty approach. We can not do it so.